### PR TITLE
Bugfix1c

### PR DESCRIPTION
**Pull request by Diversus23**
### Changes

* fix(1c) fix escaped symbols "+-;():=,[]" literals
* Added `1c-light` theme a like in the IDE 1C:Enterprise 8 (for 1c)

In IDE 1C:Enterprise 8:
image

New theme `1c-light`:
image

### Checklist

- [x] Update markup tests
- [X] Updated the changelog at `CHANGES.md`

### CODE REVIEW

File: src/styles/1c-light.css [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+  Website: https://softonit.ru/
+  License: see project LICENSE
+  Touched: 2023
+*/
+
+pre code.hljs {
+    display: block;
+    overflow-x: auto;
+    padding: 1em;
+  }
```
Comment (joshgoebel) [inline src/styles/1c-light.css:15]: Removed, this is added by the build process.
```suggestion
```

File: src/styles/1c-light.css [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+  
+  code.hljs {
+    padding: 3px 5px;
+  }
```
Comment (joshgoebel) [inline src/styles/1c-light.css:19]: Ditto.
```suggestion
```

File: src/styles/1c-light.css [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+  /* end baseline CSS */
+  
+  .hljs {
+    font-family: "Courier New", monospace; /* This font is used in IDE the 1C:Enterprise 8 */
+    color: #0000ff;
+    background: #ffffff;
+    font-weight: normal;
+    font-style: normal;
+  }
```
Comment (joshgoebel) [inline src/styles/1c-light.css:28]: There is no need for these `normal` resets and our themes are not allowed to change the font.

File: src/styles/1c-light.css (hunk 1 above)
Comment (Diversus23) [reply src/styles/1c-light.css]: Thanks for the review. Done

File: src/styles/1c-light.css (hunk 2 above)
Comment (Diversus23) [reply src/styles/1c-light.css]: Done

File: src/languages/1c.js [hunk 4]
```diff
@@ -443,6 +443,12 @@ export default function(hljs) {
     ]
   };
 
+  const PUNCTUATION = {
+    match: /[;|(|)|+|\-|:|=|,]/,
```
Comment (joshgoebel) [inline src/languages/1c.js:447]: What are all the `|` for... `|` is just another character [with no special meaning] inside a `[]` block... I think you want to just remove them all?  Or have it only a single time if `|` itself is also punctuation...

**Comment by joshgoebel** [comment]
Looking better, almost there!

File: src/languages/1c.js (hunk 4 above)
Comment (Diversus23) [reply src/languages/1c.js]: > What are all the `|` for... `|` is just another character [with no special meaning] inside a `[]` block... I think you want to just remove them all? Or have it only a single time if `|` itself is also punctuation...

You're right Josh. This is a mistake. Deleted it. Thanks!

**Review by joshgoebel — APPROVED** [review]

**Pull request merged**